docs/Deploy/Reference-Architectures.md

```diff
@@ -2,10 +2,10 @@
 
 ## The Fleet binary
 
-The Fleet application contains two single static binaries which provide web based administration, REST API, and CLI interface to Fleet.
+The Fleet application contains two single static binaries which provide web based administration, REST API, and [CLI interface](https://fleetdm.com/docs/using-fleet/fleetctl-cli) to Fleet.
```

Comment:
@JoGSal Sorry about this, the page that this links to has been moved to the guides section of the website. Can you update this URL?

```suggestion
The Fleet application contains two single static binaries which provide web based administration, REST API, and [CLI interface](https://fleetdm.com/guides/fleetctl) to Fleet.
```

Reply:
Absolutely, thank you for pointing it out!